src/EncompassRest/Loans/RateLocks/Investor.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Collections.Generic;
+
+namespace EncompassRest.Loans.RateLocks
+{
+    /// <summary>
+    /// Rate Lock Investor
+    /// </summary>
+    public sealed partial class Investor : DirtyExtensibleObject
```

Comment:
I'll take your word for it since this isn't documented in Dev Connect.

Reply:
All the properties were inferred from observed results. I'm sure there are more properties that weren't populated from my sample loans, which I would have missed.